noir-projects/aztec-nr/aztec/src/encrypted_logs/encrypted_note_emission.nr

```diff
@@ -14,7 +15,7 @@ fn compute_raw_note_log<Note, let N: u32>(
     ovpk: OvpkM,
     ivpk: IvpkM,
     recipient: AztecAddress,
-) -> (u32, [u8; 416 + N * 32], Field)
+) -> (u32, [u8; 417 + N * 32], Field)
```

Comment:
This change is in place because now we have 1 more byte in the note log for the num public values.